common/data/oauth.js

```diff
@@ -12,6 +12,17 @@ const getJwtToken = () => {
   return postData(`${url}/token?grant_type=client_credentials`)
 }
 
+const getToken = async token => {
```

Comment:
should we maybe rename this as verifyToken or something a bit more descriptive of what this endpoint does? 
are we verifying the token and getting a new one if not valid?

Reply:
Totally agree - I've just updated the naming here to be clear with the intention